src/web/Html.tsx

```diff
@@ -104,38 +133,78 @@ export const Html = React.forwardRef(
         }
       }
       // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [target])
+    }, [target, transform])
 
     const styles: React.CSSProperties = React.useMemo(
       () => ({
         position: 'absolute',
-        transform: center ? 'translate3d(-50%,-50%,0)' : 'none',
-        ...(fullscreen && {
-          top: -size.height / 2,
-          left: -size.width / 2,
+        transform: center && !transform ? 'translate3d(-50%,-50%)' : 'none',
```

Comment:
I think we should expand on this to make it easier to read:

```ts
const styles: React.CSSProperties = React.useMemo(() => {
  if(transformed){
    // code
  }
  else {
    // code
  }
})
```

I appreciate there will be some duplication, but it'll be easier to read which I think is more beneficial.

Reply:
ok, much cleaner